peachjam_subs/models.py

```diff
@@ -165,15 +165,26 @@ def __str__(self):
     @classmethod
     def product_offerings_available_to_user(cls, user):
         """Return a queryset of ProductOffering objects available to the user."""
-        return (
-            get_objects_for_user(user, "peachjam_subs.can_subscribe", klass=cls)
-            .exclude(
-                # exclude currently active subscription
+        qs = cls.objects.all()
```

Comment:
If the user isn't authenticated, then use `SubscriptionSettings.key_products` and each key product's `default_offering`.